src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -123,53 +142,69 @@ function AnnotationShareControl({
+            // IconButtons in touch interfaces
+            'absolute bottom-8 right-1 touch:bottom-touch-minimum',
+            'space-y-2 p-2',
+            // Cards do not have a border in the clean theme. Turn it back on.
+            'theme-clean:border'
+          )}
+        >
+          <div className="flex items-center pb-2 border-b">
+            <h2 className="text-brand text-lg font-medium">
               Share this annotation
-            </div>
+            </h2>
+          </div>
+          <div
+            className={classnames(
+              // Slightly larger font size for touch devices to correspond with
+              // larger button and input sizes
+              'flex w-full text-sm touch:text-base'
+            )}
+          >
+            <TextInputWithButton>
+              <TextInput
+                aria-label="Use this URL to share this annotation"
+                type="text"
+                value={shareUri}
+                readOnly
+                inputRef={inputRef}
+              />
+              <IconButton
+                icon="copy"
+                title="Copy share link to clipboard"
+                onClick={copyShareLink}
+                variant="dark"
+              />
+            </TextInputWithButton>
           </div>
-          <div className="annotation-share-panel__content">
-            <div className="hyp-u-layout-row annotation-share-panel__inputs">
-              <TextInputWithButton>
-                <TextInput
-                  aria-label="Use this URL to share this annotation"
-                  type="text"
-                  value={shareUri}
-                  readOnly
-                  inputRef={inputRef}
-                />
-                <IconButton
-                  icon="copy"
-                  title="Copy share link to clipboard"
-                  onClick={copyShareLink}
-                  variant="dark"
-                />
-              </TextInputWithButton>
-            </div>
+          <div className="text-base font-normal" data-testid="share-details">
             {inContextAvailable ? (
-              <div className="annotation-share-panel__details">
-                {annotationSharingInfo}
-              </div>
+              <>{annotationSharingInfo}</>
             ) : (
-              <div className="annotation-share-panel__details">
+              <>
                 This annotation cannot be shared in its original context because
                 it was made on a document that is not available on the web. This
                 link shares the annotation by itself.
-              </div>
+              </>
             )}
-            {showShareLinks && <ShareLinks shareURI={shareUri} />}
           </div>
-          <Icon name="pointer" classes="annotation-share-panel__arrow" />
-        </div>
+          {showShareLinks && <ShareLinks shareURI={shareUri} />}
+          <MenuArrowDown classes="bottom-[-12px] right-1 touch:right-[9px]" />
```

Comment:
In this case, I've split out what might be a reusable pattern as `MenuArrowDown` and passed the positioning classes specific to this _usage_ of a down-pointing arrow as props. Helps us later if we want to extract that `MenuArrowDown` component as a reusable presentational component.